frontend/src/slices/app/actions.ts

```diff
@@ -0,0 +1,17 @@
+import { createAsyncThunk } from '@reduxjs/toolkit';
+
+import { type NotificationPayload } from '#libs/packages/notification/libs/types/types.js';
```

Comment:
```suggestion
import { type NotificationPayload } from '#libs/packages/notification/notification.js';
```
Please, check other imports as well